rainier-docs/src/main/tut/tour.md

```diff
@@ -103,21 +105,22 @@ val sales = List(4, 4, 7, 11, 8, 12, 10)
 
 We can model the number of sales we get on each day as a poisson distribution parameterized by the underlying rate. For example, looking at that data, we might guess that it came from a `Poisson(9)` or `Poisson(10)`. We can test those hypotheses by using the `fit` method available on all `Distribution` objects. Since `Poisson` is a `Distribution[Int]`, its `fit` will accept either `Int` or `Seq[Int]`, and return a `RandomVariable` which encodes the likelihood of the provided data being produced by that distribution. (We're not going to worry right now about what *kind* of `RandomVariable` it is, since we're only really interested in the likelihood).
 
-```tut
+```tut:silent
 val poisson9: RandomVariable[_] = Poisson(9).fit(sales)
 val poisson10: RandomVariable[_] = Poisson(10).fit(sales)
 ```
 
-Although it's almost never necessary, we can reach into any `RandomVariable` and get its current probability `density`:
+A `RandomVariable`'s density is a rather opaque object and, since it is almost never necessary to reach into a `RandomVariable` and get its current probability `density`, Rainier makes this hard to do. Taking on faith momentarily that
```

Comment:
I'll try that

Reply:
@avi-stripe I pushed something, but since for the docs the point is just that you get either a Constant or something opaque, I wonder if densityValue should just return `targetGroup.base`.

Reply:
Okay, decided to just return the `base` for now.